### ISSUE TITLE
Add ability to change Python version

### ISSUE THREAD

**Issue by davidebbo** [Azure/azure-functions-host#519]
If you go to the App Settings, you can change the Python version of the Web App, but that doesn't work at all and instead completely breaks the function app. That's because this setting is not designed for Functions at all.

Instead, with Functions, it uses whatever Python.exe is on the path (see [code](https://github.com/Azure/azure-webjobs-sdk-script/blob/6539f3b370b316e44bca995b512afdb675cdfc18/src/WebJobs.Script/Description/Script/ScriptFunctionInvoker.cs#L58)), which happens to be `D:\Python27\python.exe`.

We should provide a proper way of changing this.

**Comment by epaprat** [comment]
Actulaly we solved this issue with siteextensions.

Added to our `template.json`:

```json
"resources": [
{
    "apiVersion": "2016-03-01",
    "name": "python2712x64",
    "type": "siteextensions",
    "properties": { },
    "dependsOn": [
        "[resourceId('Microsoft.Web/sites', parameters('siteName'))]"
     ]
 }
````

```json
"HandlerMappings":[{
    "Arguments":"D:\\home\\Python27\\wfastcgi.py",
    "Extension":"fastCgi",
    "ScriptProcessor":"D:\\home\\Python27\\python.exe"
 }]
```

This way is pretty hacky but it works until this issue is solved.

**Comment by davidebbo** [comment]
@epaprat Nice! Thanks for sharing.

**Comment by christopheranderson** [comment]
won't be fixed in v1, will be addressed in 2.0

**Comment by asavaritayal** [comment]
We will no longer be adding new features to Functions 1.0.

Python support has been reintroduced on Functions 2.0 and `python 3.6` is supported. You can follow the discussion here - https://aka.ms/functions-python-preview
